Week3/hw/app13.js

```diff
@@ -0,0 +1,3 @@
+//  13. Create an object that contains the teachers that you have had so far for the different modules.
+
+let seenInstructors = ['Philipp', 'Rob', 'Unmesh', 'Bonan', 'Yash'];
```

Comment:
If you combine this with the code on `line 5`, I don't think it will work properly. Because here you instantiate an `array` and then you add a `property` to that array. It's a better idea to create an `object` and then add the `instructors` as an `array property` to that object. Can you change that?